Sources/Prototyper/Feedback/EditScreenshotView.swift

```diff
@@ -35,101 +34,131 @@ struct EditScreenshotView: View {
                 Image(uiImage: self.state.screenshot!)
                     .resizable()
                     .scaledToFit()
-                    .background(RectGetter(rect: self.$rect))
                     .shadow(color: Color.primary.opacity(0.2), radius: 5.0)
-                Group {
-                    ForEach(self.allDrawings) { drawing in
-                        drawing.path
-                    }
-                    currentDrawing?.path
-                }.offset(y: -self.rect.origin.y)
-            }.gesture(dragGesture)
+                    .overlay(DrawingBoard(canvas: $canvas, isDraw: $isDraw, type: $type, color: $color))
+            }
             Spacer(minLength: 16)
             actions
-        }.onAppear(perform: setupCurrentDrawings)
-            .navigationBarTitle("Markup")
-            .navigationBarItems(leading: cancelButton, trailing: saveButton)
-            .sheet(isPresented: $colorPickerShown) {
-                NavigationView {
-                    ColorPickerView(color: self.$color, colorPickerShown: self.$colorPickerShown)
-                        .navigationBarItems(leading: self.cancelButton)
-                }
-            }
+        }.navigationBarTitle("Markup")
+        .navigationBarItems(leading: cancelButton, trailing: saveButton)
+        .sheet(isPresented: $colorPicker) {
+            ColorPicker("Pick Color", selection: $color)
+                .padding()
+        }
```

Comment:
Please try to indent modifier one level further than the modified content. This might not be consistent across the codebase but we should strive to that.
```suggestion
        }
            .navigationBarTitle("Markup")
            .navigationBarItems(leading: cancelButton, trailing: saveButton)
            .sheet(isPresented: $colorPicker) {
                ColorPicker("Pick Color", selection: $color)
                    .padding()
            }
```